mobileChatExamples/androidChatExample/README.md

```diff
@@ -1,20 +1,25 @@
 # Android Native Chat Demo 📱
 
-A native android example app for building custom Amazon Connect Chat. This solution implements basic [ChatWidget](https://docs.aws.amazon.com/connect/latest/adminguide/add-chat-to-website.html) functionality and is capable of Interactive Messages.
+This is an example app on how to utilise [AmazonConnectChatAndroid](https://github.com/amazon-connect/amazon-connect-chat-android) SDK
+
 
 > Refer to [#Specifications](#speficications) for details on compatibility, supported versions, and platforms.
 
 **Reference:**
 
 - Documentation: https://docs.aws.amazon.com/connect/latest/adminguide/enable-chat-in-app.html
```

Comment:
Should we update this to link to mobile admin guide page?

https://docs.aws.amazon.com/connect/latest/adminguide/integrate-chat-with-mobile.html

Reply:
We could also refer them to the Android SDK readme as well